examples/spot_local_order_book.rs

```diff
@@ -0,0 +1,144 @@
+use bybit::spot::ws::{OrderBookItem, PublicResponse, PublicWebSocketApiClient};
+use std::io::{self, Write};
+
+struct OwnedOrderBookItem(String, String);
+
+fn main() {
+    let mut client = PublicWebSocketApiClient::new("wss://stream.bybit.com/spot/quote/ws/v1");
```

Comment:
This example is a great addition!

I wonder if we can improve the experience of manually supplying URIs like `wss://stream.bybit.com/spot/quote/ws/v1`. Every time I need to check the docs for an endpoint I end up fiddling with a few different URIs until I find the right one. Maybe something like an enum, or is the exact URI redundant since the user has already specified `PublicWebSocketApiClient`?

Reply:
I agree. A good pattern I think is provide reasonable defaults for every `WebSocketApiClient` and allows user provide custom config to override defaults. I'll work on this.